diagnostic.md

```diff
@@ -57,13 +69,13 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+$('li').parentsUntil('body').hide()
 <!-- your answer ends here -->
 
 ## Question 8
 
 Use jQuery to register a click handler for clicks inside `div` items.
 
 <!-- your answer starts here -->
-
+$('div').children().on('click')
```

Comment:
this is almost there, you just need to pass a function as the second argument so it knows what to run when you click the div.